### PR TITLE
Feature/erp invoice

### PR DESCRIPTION
**Pull request by julianzimmermann**

### CODE REVIEW

File: bundles/erp-invoice-api/src/FondOfOryx/Shared/ErpInvoiceApi/Transfer/erp_invoice_api.transfer.xml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+<transfers xmlns="spryker:transfer-01"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="spryker:transfer-01 http://static.spryker.com/transfer-01.xsd">
+    <transfer name="ErpInvoice">
+        <property name="idErpInvoice" type="int"/>
+    </transfer>
+
+    <transfer name="ErpInvoiceResponse">
+        <property name="isSuccessful" type="bool"/>
+        <property name="erpInvoice" type="ErpInvoice"/>
```
Comment (daniel-rose) [inline bundles/erp-invoice-api/src/FondOfOryx/Shared/ErpInvoiceApi/Transfer/erp_invoice_api.transfer.xml:11]: Property "message" is missing.

Comment (daniel-rose) [inline bundles/erp-invoice-api/src/FondOfOryx/Shared/ErpInvoiceApi/Transfer/erp_invoice_api.transfer.xml:11]: Property "message" is missing.

File: bundles/erp-invoice-api/src/FondOfOryx/Zed/ErpInvoiceApi/ErpInvoiceApiDependencyProvider.php [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+     * @param \Spryker\Zed\Kernel\Container $container
+     *
+     * @return \Spryker\Zed\Kernel\Container
+     */
+    public function provideBusinessLayerDependencies(Container $container): Container
+    {
+        $container = parent::provideBusinessLayerDependencies($container);
+
+        $container = $this->addErpInvoiceFacade($container);
+        $container = $this->addApiQueryContainer($container);
```
Comment (daniel-rose) [inline bundles/erp-invoice-api/src/FondOfOryx/Zed/ErpInvoiceApi/ErpInvoiceApiDependencyProvider.php:44]: Persistence class in business layer?

File: bundles/erp-invoice/src/FondOfOryx/Zed/ErpInvoice/Communication/Console/ErpInvoiceConsole.php [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+use Spryker\Zed\Kernel\Communication\Console\Console;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * @method \FondOfOryx\Zed\ErpInvoice\Business\ErpInvoiceFacadeInterface getFacade()
+ * @method \FondOfOryx\Zed\ErpInvoice\Persistence\ErpInvoiceRepositoryInterface getRepository()
+ */
+class ErpInvoiceConsole extends Console
```
Comment (daniel-rose) [inline bundles/erp-invoice/src/FondOfOryx/Zed/ErpInvoice/Communication/Console/ErpInvoiceConsole.php:19]: Maybe "ErpInvoicePlaygroundConsole" will be a better name for this?

File: bundles/erp-invoice/src/FondOfOryx/Zed/ErpInvoice/Exception/WrongInterfaceException.php [hunk 4]
```diff
@@ -0,0 +1,9 @@
+<?php
+
+namespace FondOfOryx\Zed\ErpInvoice\Exception;
+
+use Exception;
+
+class WrongInterfaceException extends Exception
```
Comment (daniel-rose) [inline bundles/erp-invoice/src/FondOfOryx/Zed/ErpInvoice/Exception/WrongInterfaceException.php:7]: What do you mean with wrong interface?

**Review by daniel-rose — APPROVED** [review]
transfer definition is missing in erp-invoice-extension

**Review by gengjozsef — APPROVED** [review]

**Pull request merged**